Looms.PoS/Looms.PoS.Application/Models/Requests/PaymentProvider/CreatePaymentProviderRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Looms.PoS.Domain.Enums;
+
+namespace Looms.PoS.Application.Models.Requests.PaymentProvider;
+
+public record CreatePaymentProviderRequest
+{
+    public string Name { get; init; } = string.Empty;
+    public PaymentProviderType Type { get; init; }
```

Comment:
what error happens if you provide a type that is not in enum?

Reply:
validation fails